### ISSUE TITLE
can someone port nvidia/parakeet-rnnt-1.1b

### ISSUE THREAD

**Issue by Kreijstal** [ggml-org/whisper.cpp#1732]
I heard it is super fast, way faster than whisper

**Comment by bobqianic** [comment]
I don't think it will be as robust as Whisper. Its dataset is an order of magnitude smaller than Whisper's and it only supports English, which, according to research from ~Open~CloseAI, leads to a higher WER.

**Comment by nithinraok** [comment]
Its actually pretty robust and more importantly no hallucinations. 
It only currently supports English, but gets very strong numbers on English benchmarks as can be seen [here](https://huggingface.co/spaces/hf-audio/open_asr_leaderboard).
More info: https://arxiv.org/abs/2305.05084

**Comment by smeett247** [comment]
my question is which api PARAKEET_URL = "http://localhost:9000/speech-to-text" instead of this what is the correct URL

**Comment by nithinraok** [comment]
would you like to try out the model or to serve in your production use case?
